R/tcplConf.R

```diff
@@ -71,6 +71,15 @@ tcplConf <- function (drvr = NULL, user = NULL, pass = NULL, host = NULL,
       options("TCPL_DRVR" = "tcplLite")
     } 
     
+    if (drvr == "API") {
+      options("TCPL_DRVR" = "API")
+      if (is.null(pass)) stop("'API' driver requires an API-key, supply it to 
+                              the 'pass' parameter. To request a key, send an
+                              email to ccte_api@epa.gov.")
+      if (is.null(host)) options("TCPL_HOST" = "https://api-ccte.epa.gov/bioactivity")
+      register_ccdr(key = pass)
```

Comment:
calling register_ccdr requires @import statement in roxygen skeleton/documentation

Reply:
purposefully haven't added ccdR imports as they would would make it so you couldn't load the package i thought. Waiting until ccdR is on CRAN